### DISCUSSION TITLE
Disable the question

### DISCUSSION THREAD

**Discussion by richardihm** [chuongmep/RevitAddInManager#51] [Ideas]
Hello,
is there a way to disable the question for large Folder projects wen i run a comand? I allways like to copy all files to the temp folder.

thx. 
![Unbenannt](https://github.com/chuongmep/RevitAddInManager/assets/152442005/fb0208be-1a25-43a8-934a-683ae65e449d)

**Comment by chuongmep** [comment]
Hi @richardihm , at the moment , we don't have any feature to configuration that, may be can consider improvement in feature.
